lib/rake/baseextensiontask.rb

```diff
@@ -35,7 +35,7 @@ def initialize(name = nil, gem_spec = nil)
     end
 
     def init(name = nil, gem_spec = nil)
-      @name = name
+      @name = name.to_s
```

Comment:
> This change works only for Task.new(name) case.
> We need to care about Task#name= case.
>
> How about stringifying in #define?

Thank you for nice suggestion. Fixed to stringify name in `#define`. https://github.com/rake-compiler/rake-compiler/pull/134/commits/af7dee631b4011b9f2255c8c83c3ee8ae984c374